### PR TITLE
019-form2.1

### PR DESCRIPTION
**Pull request by FrancoisSalmon**
…o receive message, add and configure toastify

### CODE REVIEW

**Review by Gonaline — DISMISSED** [review]
ok

**Review by LuigiMenez — DISMISSED** [review]
Ok for me.

**Review by Gonaline — APPROVED** [review]
ok

**Review by LuigiMenez — APPROVED** [review]
Ok for me.

**Comment by LoicBrassart** [comment]
Conflicts still

**Review by LuigiMenez — CHANGES_REQUESTED** [review]
Fix conflict

**Comment by LoicBrassart** [comment]
Conflicts + Plz stop asigning me, Luigi is in charge of the 2nd review

**Review by Gonaline — DISMISSED** [review]
ok

File: frontend/src/services/context/Ctx.jsx [hunk 1]
```diff
@@ -12,6 +12,7 @@ export function CtxProvider({ children }) {
   const [news, setNews] = useState([]);
   const [team, setTeam] = useState([]);
 
+
```
Comment (LuigiMenez) [inline frontend/src/services/context/Ctx.jsx:15]: Line Blank useless

File: backend/src/router.js [hunk 2]
```diff
@@ -43,6 +44,9 @@ router.put("/role/:id", RoleController.edit);
 router.post("/role", RoleController.add);
 router.delete("/role/:id", RoleController.delete);
 
+
```
Comment (LuigiMenez) [inline backend/src/router.js:47]: Line blank useless

**Review by LuigiMenez — CHANGES_REQUESTED** [review]
Line blank useless

**Review by Gonaline — APPROVED** [review]
ok

**Review by LuigiMenez — APPROVED** [review]
Ok for me.

File: frontend/src/components/Form/index.jsx [hunk 3]
```diff
@@ -1,20 +1,74 @@
...
+  const hSubmit = (evt) => {
+    evt.preventDefault();
+    axios
+      .post(`${import.meta.env.VITE_BACKEND_URL}/contact`, formData)
+      .then(() => {
+        toast("Message envoyé!");
+      })
+      .catch(() => {
+        toast.error(
+          "Un problème est survenue, merci de réessayer ultérieurement "
```
Comment (LoicBrassart) [inline frontend/src/components/Form/index.jsx:25]: "survenu"

**Review by LoicBrassart — APPROVED** [review]
SnE, one typo to fix

**Pull request merged**